mk/xe-linux-distribution

```diff
@@ -148,6 +149,7 @@ identify_redhat()
                -e 's/^CentOS release \([0-9]*\)\.\([0-9]*\) (.*)/distro=centos;major=\1;minor=\2/gp;' \
                -e 's/^CentOS release \([0-9]*\) (.*)/distro=centos;major=\1/gp;' \
                -e 's/^CentOS Linux release \([0-9]*\)\.\([0-9]*\).*$/distro=centos;major=\1;minor=\2/gp;' \
+               -e 's/^CentOS Stream release \([0-9]*\).*$/distro=centos;major=\1/gp;' \
```

Comment:
Thanks for your contribution! 
LGTM, BTW have you tested this change on the CentOS Stream guest already?

Reply:
> have you tested this change on the CentOS Stream guest already

Sure have. After this change, `/var/cache/xe-linux-distribution` is created (where it was not before) and includes the correct information, so `xe-daemon` works properly and stops reporting `CollectOS` errors.

The catch-all portion at the end of the expression is currently unneeded as there's no characters after the version number, but I included it in case Red Hat decided to add a line-break or something for some reason later down the road.

Reply:
is the centos stream reality now?

Reply:
@Zhengchai Red Hat is getting rid of CentOS Linux and pushing Stream as the replacement.

[Link](https://blog.centos.org/2020/12/future-is-centos-stream/)